utils/build.ps1

```diff
@@ -175,6 +175,12 @@ $msbuild = "$VSInstallRoot\MSBuild\Current\Bin\$BuildArchName\MSBuild.exe"
 # Avoid $env:ProgramFiles in case this script is running as x86
 $UnixToolsBinDir = "$env:SystemDrive\Program Files\Git\usr\bin"
 
+# The make tool isn't part of MSYS, but we need it for LLDB tests
+$MakeToolPath = "C:/Program Files (x86)/GnuWin32/bin/make.exe"
+if (-not (Test-Path $MakeToolPath)) {
+  & winget install GnuWin32.Make
```

Comment:
Alternatively, we could also use https://scoop.sh/ and run `scoop install main/make`. This is what the upstream bot https://lab.llvm.org/buildbot/#/builders/141 does and they dump:
```
-- Found make: C:/Users/tcwg/scoop/shims/make.exe
```